perf_testing/scripts/highspeed_create.py

```diff
@@ -29,7 +29,7 @@ def main(folder, num_files):
     total_data_written = num_files * 20  # in GB
     speed_gbps = (total_data_written * 8) / total_time  # converting GB to Gb and then calculating Gbps
 
-    print(json.dumps({"name": "create_10_20GB_file", "total_time": total_time, "speed": speed_gbps, "unit": "GiB/s"}))
+    print(json.dumps({"name": "create_10_20GB_file", "total_time": total_time, "speed": speed_gbps / 8, "unit": "GiB/s"}))
```

Comment:
This will invalidate the current values in the graph and we might have to cleanup all old data. How do you want to clean those up.

Reply:
Thanks for the heads up, will find a way to do that

Reply:
We might need to delete all of old data before making this change. Let's clean it up just before merging this.